lib/modules/manager/nuget/config-formatter.spec.ts

```diff
@@ -0,0 +1,271 @@
+import { mockDeep } from 'jest-mock-extended';
+import { XmlDocument } from 'xmldoc';
+import { mocked } from '../../../../test/util';
+import * as _hostRules from '../../../util/host-rules';
+import { createNuGetConfigXml } from './config-formatter';
+import type { Registry } from './types';
+
+jest.mock('../../../util/host-rules', () => mockDeep());
+
+const hostRules = mocked(_hostRules);
```

Comment:
I would prefer to not mock hostrule and instead clearing them in beforeEach

Reply:
Done, see https://github.com/renovatebot/renovate/pull/25052/commits/b3aba7269bac27eded5b6130856c99e28c8c9601